configure.ac

```diff
@@ -595,7 +595,7 @@ AS_IF([test x"$with_cuda" != x"no"],
 				 [cuda_runtime.h],
 				 [cudart],
 				 [cudaMemcpy],
-				 [-lcuda],
+				 [-lcuda -lnvidia-ml],
```

Comment:
What is the impact of requiring this new library?  Is it always available?

There's a missing set of checks somewhere, since we now need nvml.h present.

Reply:
Yes, the nvml header is installed with CUDA, since 2012.  I don't think we need another check for it.